CHANGES.rst

```diff
@@ -43,6 +43,9 @@ astropy.io.misc
 
 - Eliminate deprecated compatibility mode when writing ``Table`` metadata to HDF5 format. [#8899]
 
+- Add support for orthogonal polynomial models to ASDF [requires
+  asdf-standard PR #204]
```

Comment:
Put in brackets the PR number, `[#9107]`,  and remove `requires
  asdf-standard PR #204`.